packages/itwinui-css/src/tile/tile.scss

```
@@ -353,19 +352,24 @@ $thumbnail-selector: ':is(.iui-thumbnail-icon, .iui-tile-thumbnail-picture)';
     justify-self: end;
   }
 
-  .iui-tile-thumbnail-picture ~ & {
-    @include mixins.iui-blur($opacity: 5);
-    --_iui-button-icon-fill: var(--iui-color-white);
+  .iui-tile-thumbnail-button {
+    border-radius: 50%;
```

```suggestion
    border-radius: var(--iui-border-radius-round);
```

Feel like this should be changed, but will likely effect visuals tests.  If we like this change, I'll make it right at the end to keep image test changes until the last push.